ios/brave-ios/Sources/Brave/Frontend/Settings/Display/NTPTableViewController.swift

```diff
@@ -84,21 +90,41 @@ class NTPTableViewController: TableViewController {
     dataSource.sections = [imageSection, widgetSection]
   }
 
-  private func selectedItem() -> BackgroundImageType {
+  private func selectedItem() -> BackgroundType {
     if let referrer = Preferences.NewTabPage.selectedCustomTheme.value {
       return .superReferrer(referrer)
     }
 
-    return Preferences.NewTabPage.backgroundSponsoredImages.value ? .sponsored : .defaultImages
+    switch Preferences.NewTabPage.backgroundMediaType {
+    case .defaultImages:
+      return BackgroundType.defaultImages
+    case .sponsoredImages:
+      return BackgroundType.sponsoredImages
+    case .sponsoredImagesAndVideos:
+      return rewards?.shouldShowSponsoredImagesAndVideos == true
+        ? BackgroundType.sponsoredImagesAndVideos(
+          Strings.NTP.settingsSponsoredImagesAndVideosSelection
+        )
+        : BackgroundType.sponsoredImagesAndVideos(Strings.NTP.settingsSponsoredImagesSelection)
+    }
   }
 
-  private lazy var backgroundImageOptions: [BackgroundImageType] = {
-    var available: [BackgroundImageType] = [.defaultImages, .sponsored]
+  private func backgroundImageOptions() -> [BackgroundType] {
+    var available: [BackgroundType] = [.defaultImages]
+    if rewards?.shouldShowSponsoredImagesAndVideos == true {
+      available += [
+        .sponsoredImages,
+        .sponsoredImagesAndVideos(Strings.NTP.settingsSponsoredImagesAndVideosSelection),
+      ]
+    } else {
+      available += [.sponsoredImagesAndVideos(Strings.NTP.settingsSponsoredImagesSelection)]
```

Comment:
Wouldn't this just be `sponsoredImages`?

Reply:
For countries, which doesn't support NTT video we show `Sponsored Images` option but under the hood it is  `.sponsoredImagesAndVideos`. 
This will allow us to avoid preferences migration issues when NTT video is available for all countries. 

So to add support for all countries with this approach, we basically just need to get rid of `shouldShowSponsoredImagesAndVideos` function only.